### ISSUE TITLE
Fail to run the test driver of devenc

### ISSUE THREAD

**Issue by yuhow** [solvcon/devenv#133]
The output message are listed below: 
---------------------------------------------------------------------------------
*** test file: test_init_var.sh
test_devenvroot
test_devenvflavor
test_devenvflavorroot
test_devenvcurrentroot
test_devenvlibrary_path_backup
test_devenvpath_backup
test_devenvprefix
test_devenvdlroot

Ran 8 tests.

OK
*** test file: test_bash_utils.sh
test_display_type
ASSERT:expected:<display is a function> but was:<>
shunit2:ERROR test_display_type() returned non-zero return code.
test_display
test_display_e
devenv shunit2 unittest
test_get_cmd_list
test_get_build_list
test_get_application_list
test_get_flavor_list

Ran 7 tests.

FAILED (failures=2)
*** test file: test_devenv_impl.sh
test_namemunge
test_nameremove

Ran 2 tests.

OK

---------------------------------------------------------------------------------

There is a conflict between the function 'display()' of devenv and the shell command /usr/bin/display.
The display is a command provided by the package 'imagemagick'.
Possibly we should change the function name of 'display()' of devenv.

**Comment by yungyuc** [comment]
Agree.  `display` is a very generic name and the build system should yield. If we cannot think of a better verb for the helper, perhaps a prefix like `devenv_display` can be used.
